libs/ui/atoms/src/lib/Icon/Icon.tsx

```diff
@@ -80,126 +80,139 @@ enum IconType {
 }
 
 interface IIconProps {
+  size?: number | undefined;
+  color?: string | undefined;
+  backgroundColor?: string | undefined;
+  borderRadius?: number | undefined;
+  onPress?: void;
   name: IconType;
+  iconStyle?: object | undefined;
 }
 
-const size = 17;
-
 function Icon(props: IIconProps) {
+  const iconProperties = {
+    size: props.size || 20,
+    color: props.color || '#009688',
```

Comment:
Could we use colors from our color palette instead of introducing some new color? Or just use primary as default?

Reply:
We can change this default colour from our palette. This is by default from library.

Reply:
We can try